interface/backend/submission/evaluator/vmck.py

```diff
@@ -0,0 +1,71 @@
+import configparser
+from urllib.parse import urljoin
+import logging
+
+from django.conf import settings
+import requests
```

Comment:
Please keep the order:
```suggestion
import logging
import configparser
from urllib.parse import urljoin

import requests
from django.conf import settings
```